### ISSUE TITLE
Animated avatars play constantly and cannot be stopped (WCAG 2.1 / 2.2.2 violation)

### ISSUE THREAD

**Issue by Mikaela** [vector-im/hydrogen-web#841]
**Describe the bug**

https://github.com/vector-im/element-web/issues/18442 again, when you join a room where someone uses an animated avatar using a mediarepo/homeserver with support for that, they play constantly and cannot be stopped.

* [Understanding Success Criterion 2.2.2: Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html)

**To Reproduce**

1. Use homeserver that plays animated avatars such as `the-apothecary.club` or `feline.support`.
2. Join a room that contains animated avatar or DM `@cat:feline.support`
3. Observe the avatar playing constantly.

**Expected behavior**

For accessibility reasons, there should be a method of stopping the avatar. Personally I am on autism spectrum and animated avatars negatively impair my ability to focus on reading or writing text making Hydrogen not great for communication.

**Screenshots**

I don't know how to animate screenshots, please refer to previous link.

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser Safari
 -  - Version 16.0 (20A5339d)

**Smartphone (please complete the following information):**
 - Device: iPhone SE2020
 - OS: iOS 16.0 (20A5339d)
 - Browser Safari
 - Version 16.0 (20A5339d)

**Additional context**

The web content accessibility guidelines expain this better than I can.

* https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html

I use iOS Accessibility features regarding motion and decrease motion and prefer "ristihäivytyssiirto", disable message effects and disable playing of video previews.
